js/editor-libs/console.js

```diff
@@ -13,8 +13,8 @@ module.exports = function() {
     };
 
     // eslint-disable-next-line no-console
-    console.log = function(loggedItem) {
-        consoleUtils.writeOutput(consoleUtils.formatOutput(loggedItem));
+    console.log = function() {
+        consoleUtils.writeOutput(consoleUtils.concatOutput(arguments));
```

Comment:
Perhaps the context is making this a little blurry but, it seems that this change will mean we loose the formatting done by `formatOutput`. That is definitely something we do not want to happen. Perhaps we should instead loop over the arguments, passing each one to the format function before concatenating together?